lib/portlayer/exec/container.go

```diff
@@ -151,6 +151,20 @@ func (s State) String() string {
 	return ""
 }
 
+// CurrentState returns current state.
+func (c *Container) CurrentState() State {
```

Comment:
Why is this preferred vs accessing `State` publicly?  Are you trying to avoid torn writes or...?